app/src/main/java/com/framgia/music_44/screen/mainActivity/MainActivity.java

```diff
@@ -70,4 +104,59 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
         }
         return false;
     }
+
+    public boolean checkPermissionREAD_EXTERNAL_STORAGE(final Context context) {
```

Comment:
check convention method and access modifier
```suggestion
     private boolean checkPermissionReadExternalStorage(final Context context) {
```